### PR TITLE
feat: print_min_max

### PR DESCRIPTION
**Pull request by aniketbiprojit**

### CODE REVIEW

File: basics/basics.cpp [hunk 1]
```diff
@@ -18,6 +18,14 @@
 /** Global int */ int globalVar = 0;
 /** Constant global double */ const double PI = 3.141;
 
+template <typename T>
+void print_min_max(std::string name)
+{
+    std::cout << "Min :" << name << " > " << std::numeric_limits<unsigned short int>::min() << std::endl;
```
Comment (AbhinavRobinson) [inline basics/basics.cpp]: Amazing 👍🏽

File: basics/basics.cpp [hunk 2]
```diff
@@ -18,6 +18,9 @@
 /** Global int */ int globalVar = 0;
 /** Constant global double */ const double PI = 3.141;
 
+/**
```
Comment (AbhinavRobinson) [inline basics/basics.cpp:21]: Okay 💯

File: basics/basics.cpp [hunk 3]
```diff
@@ -24,8 +24,8 @@
 template <typename T>
 void print_min_max(std::string name)
 {
-    std::cout << "Min :" << name << " > " << std::numeric_limits<unsigned short int>::min() << std::endl;
-    std::cout << "Max :" << name << " > " << std::numeric_limits<unsigned short int>::max() << std::endl;
+    std::cout << "Min :" << name << " > " << std::numeric_limits<T>::min() << std::endl;
```
Comment (AbhinavRobinson) [inline basics/basics.cpp:27]: was wondering why it was still printing unsigned only

File: basics/basics.cpp (hunk 1 above)
Comment (aniketbiprojit) [reply basics/basics.cpp]: Outdated.

File: basics/basics.cpp (hunk 3 above)
Comment (aniketbiprojit) [reply basics/basics.cpp:27]: Lol

File: basics/basics.cpp (hunk 1 above)
Comment (aniketbiprojit) [reply basics/basics.cpp]: Pushed fixes. xD.

**Pull request merged**